openrtb_ext/bidders.go

```diff
@@ -199,7 +200,6 @@ const (
 	BidderUcfunnel          BidderName = "ucfunnel"
 	BidderUnicorn           BidderName = "unicorn"
 	BidderUnruly            BidderName = "unruly"
-	BidderValueImpression   BidderName = "valueimpression"
```

Comment:
You still need the valueimpression entries in this file for the alias.

You'd also need the `static/bidder-info/valueimpression.yaml` and `static/bidder-params/valueimpression.json` files for the alias.